diff --color -Naur node_modules/@stellar/design-system.backup/build/components/Alert/index.d.ts node_modules/@stellar/design-system/build/components/Alert/index.d.ts
--- node_modules/@stellar/design-system.backup/build/components/Alert/index.d.ts	2025-08-25 10:47:32
+++ node_modules/@stellar/design-system/build/components/Alert/index.d.ts	2025-08-25 10:47:33
@@ -1,4 +1,4 @@
-import "./styles.scss";
+
 export type AlertBaseProps = {
     variant: "primary" | "secondary" | "success" | "warning" | "error";
     placement: "inline" | "bottom";
@@ -18,4 +18,3 @@
 };
 export type AlertProps = (AlertActionButton | AlertActionLink) & AlertBaseProps;
 export declare const Alert: React.FC<AlertProps>;
-//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --color -Naur node_modules/@stellar/design-system.backup/build/components/Alert/index.js node_modules/@stellar/design-system/build/components/Alert/index.js
--- node_modules/@stellar/design-system.backup/build/components/Alert/index.js	2025-08-25 10:47:32
+++ node_modules/@stellar/design-system/build/components/Alert/index.js	2025-08-25 10:47:33
@@ -5,7 +5,7 @@
 import { Icon } from "../../icons";
 import { Link } from "../Link";
 import { NavButton } from "../NavButton";
-import "./styles.scss";
+
 export var Alert = function (_a) {
     var variant = _a.variant, placement = _a.placement, title = _a.title, onClose = _a.onClose, onAction = _a.onAction, actionLink = _a.actionLink, _b = _a.actionLabel, actionLabel = _b === void 0 ? "Action" : _b, _c = _a.icon, icon = _c === void 0 ? _jsx(Icon.InfoCircle, {}) : _c, children = _a.children;
     if (placement === "bottom" && actionLink) {
@@ -49,4 +49,3 @@
     return placement === "inline" ? (_jsx(Card, { noPadding: true, variant: variant === "secondary" ? "secondary" : "primary", children: renderAlert() })) : (_jsx("div", { className: "AlertContainer", children: renderAlert() }));
 };
 Alert.displayName = "Alert";
-//# sourceMappingURL=index.js.map
\ No newline at end of file
